CHANGELOG.md

```diff
@@ -10,6 +10,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `Int.mulmod` - combined builtin operation for multiplication and modulus.
 - `Crypto.poseidon` - a ZK/SNARK-friendly hash function (over the BLS12-381 scalar field).
 - `Address.to_bytes` - convert an address to its binary representation (for hashing, etc.).
+- Raw data pointers added to `AENS`. In short this introduces new types
+  `AENS.nameV2` and `AENS.pointeeV2`, where the latter has a constructor
+  `DataPtV2(string)`; `AENS.lookup` and `AENS.update` consume and produce the
+  new types. Standard library `AENSCompat` is added to convert between old and
+  new pointers.
```

Comment:
I don't like mixing incompatible functionalities in the same namespace. Let's create `AENS2` namespace and pack all the new stuff in there.